src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -414,7 +414,7 @@ public function testGetPreferredFormat()
         $request = new Request();
         $request->headers->set('Content-Type', 'application/json');
         $request->headers->set('Accept', 'application/xml');
```

Comment:
[optional] Could you also please reorder these lines to reflect the new priority?

Reply:
i.e. the order helps (visually) to understand better the priority of the options.

Reply:
done